sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -38,13 +38,25 @@ export interface CommunicationUserToken extends CommunicationAccessToken {
     user: CommunicationUserIdentifier;
 }
 
+// @public
+export interface CreateUserAndTokenOptions extends OperationOptions {
+    // (undocumented)
```

Comment:
instead we can describe the property with a good commend, same for line 56

Reply:
Description added.